packages/cli/generation/remote-generation/remote-workspace-runner/src/runRemoteGenerationForDocsWorkspace.ts

```diff
@@ -22,6 +23,26 @@ export async function runRemoteGenerationForDocsWorkspace({
 }): Promise<void> {
     const instances = docsWorkspace.config.instances;
 
+    console.log(docsWorkspace.config.analytics);
+
+    // Substitute templated environment variables:
+    // If the run is a preview, we'll substitute ALL environment variables as empty strings
+    //
+    // Otherwise, we'll attempt to read and replace the templated environment variable. Will
+    // bubble up an error if the env var isn't found.
+    //
+    // Although this logic is separate from generating a remote, placing it here helps us
+    // avoid making cascading changes to other workflows.
+    // docsWorkspace = substituteEnvVariables(docsWorkspace, context, { substituteAsEmpty: preview });
+    docsWorkspace = replaceEnvVariables(
+        docsWorkspace,
```

Comment:
i think you only want to do this on `docsWorkspace.config` (which is the contents of `docs.yml`)